muted-tests.yml

```diff
@@ -91,8 +91,6 @@ tests:
 - class: org.elasticsearch.search.StressSearchServiceReaperIT
   method: testStressReaper
   issue: https://github.com/elastic/elasticsearch/issues/115816
-- class: org.elasticsearch.xpack.application.connector.ConnectorIndexServiceTests
```

Comment:
The `ConnectorIndexServiceTests` has been running fine for almost a year, I saw seemingly unrelated issue in the failed buildkite run that led to muting this